lib/shared/widgets/custom_navigation_drawer.dart

```diff
@@ -235,7 +235,7 @@ class CustomNavigationDrawer extends StatelessWidget {
           ),
           if (showFamilyItem) _buildEmptySpace(),
           _buildMenuItem(
-            isVisible: showFamilyItem,
+            isVisible: false,
```

Comment:
So we still want to show it on dev environment?

Reply:
We do, fixed.